src/components/visualizations/chart/HighChartsRenderer.tsx

```diff
@@ -180,6 +182,15 @@ export default class HighChartsRenderer
                 visible
             };
         }));
+
+        yAxis.forEach((axis: Highcharts.AxisOptions) => {
+            const startOrEndOnTick = shouldStartOrEndOnTick(
+                { ...this.props.chartOptions,  data: { series: config.series } }
```

Comment:
Consider composing chartOptions outside of the for-cycle, also maybe having the inner in a function (and just pass function to the foreach `yAxis.forEach(setStartEndTick)`) would be beneficial (this method starts to grow)